gateway-service-impl/src/main/java/org/hypertrace/gateway/service/span/SpanService.java

```diff
@@ -60,7 +60,7 @@ public SpansResponse getSpansByFilter(RequestContext context, SpansRequest reque
     final Context timerContext = queryExecutionTimer.time();
     try {
       Map<String, AttributeMetadata> attributeMap =
-          attributeMetadataProvider.getAttributesMetadata(context, AttributeScope.EVENT);
+          attributeMetadataProvider.getAttributesMetadata(context, AttributeScope.EVENT.name());
```

Comment:
ditto about the enum

Reply:
Yeah this service itself is specific to spans, hence this enum is hard-coded. This doesn't come from user request directly.

Reply:
See the comment above about moving away completely from AttributeScope enum.